scripts/core/event.yml

```diff
@@ -41,25 +41,6 @@ etors:
       desc: "A 64-bit value of current device counter in nanoseconds when the event has finished execution"
 --- #--------------------------------------------------------------------------
 type: function
-desc: "Create an event object. Events allow applications to enqueue commands that wait on an event to finish or signal a command completion."
-class: $xEvent
-name: Create
-ordinal: "0"
-analogue:
-    - "**clCreateUserEvent**"
-params:
-    - type: $x_context_handle_t
-      name: hContext
-      desc: "[in] handle of the context object"
-    - type: $x_event_handle_t*
-      name: phEvent
-      desc: "[out] pointer to handle of the event object created"
-returns:      
-    - $X_RESULT_ERROR_INVALID_CONTEXT
-    - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
-    - $X_RESULT_ERROR_OUT_OF_RESOURCES
---- #--------------------------------------------------------------------------
-type: function
 desc: "Get event object information"
 class: $xEvent
 name: GetInfo
```

Comment:
should we edit x_execution_info_t as well? it has some status info about user created events.

Cant seem to get this comment on that line, its on: 209 and 211

Reply:
How does [this](https://github.com/oneapi-src/unified-runtime/compare/be9ab12ddb1d207211cbfcaf644e5d2c0857df8b..1013327ab435a178da3930d7c74c6670ae8ccf4e) look?

Reply:
LGTM